packages/_site/src/pages/components/loading/index.tsx

```diff
@@ -98,6 +98,56 @@ export default function LoadingExample() {
         <LoadingSpinner size={LoadingSize.SMALL} />
 	);
 }
+`}
+                                />
+                            </div>
+                        </div>
+                        <div>
+                            <ExampleBlock
+                                title="Default"
```

Comment:
If you're adding properties to the component than it isn't default, name it after the variant form.